contents/dataframesmeta_orderby.md

```diff
@@ -0,0 +1,40 @@
+## Row Sorting {#sec:dataframesmeta_orderby}
+
+**DFM has a macro for sorting rows: `@orderby`**.
+`@orderby` do not have an in-place or vectorized variant.
+
+Let's sort our `leftjoined` by grade in 2021:
+
+```jl
+sco("""
+@orderby leftjoined :grade_2021
+"""; process=without_caption_label
+)
+```
+
+By default `@orderby` will sort in ascending order.
```

Comment:
```suggestion
By default, `@orderby` will sort in ascending order.
```
Comma is needed because the sentence is turned around. Non-turned around version would be: "`@orderby` will sort in ascending order by default".